src/decision/location_selection.jl

```diff
@@ -0,0 +1,151 @@
+
+"""
+    site_selection(domain::Domain, scenario::DataFrameRow{DataFrame,DataFrames.Index}, w_scens::NamedDimsArray, dhw_scens::NamedDimsArray, site_ids::Vector{Int64}, sum_cover::AbstractArray, area_to_seed::Float64)
+
+Perform site selection using a chosen mcda aggregation method, domain, initial cover, criteria weightings and thresholds.
+
+# Arguments
```

Comment:
The argument `domain` is missing in this list and the argument `mcda_vars` does not exist